api/acl/tests/test_models.py

```diff
@@ -0,0 +1,9 @@
+from django.test import TestCase
+from acl.models import Acl
+
+
+class AclModelTest(TestCase):
+    def test_create_acl(self):
```

Comment:
Is this test necessary? it looks to be testing Django orm.

Reply:
Removed